scripts/generate-integration-pages.ts

```diff
@@ -178,6 +179,47 @@ function absoluteLinks({ base }: { base: string }) {
 	};
 }
 
+/** Remark plugin to replace GitHub . */
+function replaceAsides() {
+	return function transform(tree: Root) {
+		// visit blockquote
+		visit(tree, 'blockquote', (node) => {
+			// first child (<blockquote><p>...</p></blockquote>)
+			const [firstChild, trailingText] = node.children[0].children;
+
+			if (firstChild.type === 'strong') {
+				if (
+					// check for **Note** or **Warning**
+					firstChild.children[0].value.includes('Note') ||
+					firstChild.children[0].value.includes('Warning')
```

Comment:
I might be tempted to reach for a regular expression here:

```suggestion
					/Note|Warning/.test(firstChild.children[0].value)
```